test/exit.js

```diff
@@ -1,38 +1,33 @@
 var tap = require('tap');
 var spawn = require('child_process').spawn;
-var trim = require('string.prototype.trim');
+var concat = require('concat-stream');
 
 tap.test('exit ok', function (t) {
     t.plan(2);
     
-    var tc = tap.createConsumer();
-    
-    var rows = [];
-    tc.on('data', function (r) { rows.push(r) });
-    tc.on('end', function () {
-        var rs = rows.map(function (r) {
-            if (r && typeof r === 'object') {
-                return { id : r.id, ok : r.ok, name : trim(r.name) };
-            }
-            else return r;
-        });
-        t.same(rs, [
+    var tc = function (rows) {
+        t.same(rows.toString('utf8'), [
             'TAP version 13',
-            'array',
-            'hi',
-            { id: 1, ok: true, name: 'should be equivalent' },
-            { id: 2, ok: true, name: 'should be equivalent' },
-            { id: 3, ok: true, name: 'should be equivalent' },
-            { id: 4, ok: true, name: 'should be equivalent' },
-            { id: 5, ok: true, name: 'should be equivalent' },
-            'tests 5',
-            'pass  5',
-            'ok'
-        ]);
-    });
+            '# array',
+            '# hi',
+            'ok 1 should be equivalent',
+            'ok 2 should be equivalent',
+            'ok 3 should be equivalent',
+            'ok 4 should be equivalent',
+            'ok 5 should be equivalent',
+            '',
+            '1..5',
+            '# tests 5',
+            '# pass  5',
+            '',
+            '# ok',
+            '',
+            ''
```

Comment:
@ljharb yeah, this is already present in `master` see: [`test/double_end.js#L25`](https://github.com/substack/tape/blob/092344b906cd3774ba1812d2db431e77916c9f19/test/double_end.js#L25) if I update the `test/exit.js` to _match_ this will that be OK? ❓